malwarefront/config-overrides.js

```diff
@@ -1,7 +1,76 @@
-const { override, babelInclude, removeModuleScopePlugin } = require('customize-cra');
+const { override, babelInclude, removeModuleScopePlugin, addWebpackPlugin, addWebpackModuleRule, addWebpackResolve } = require('customize-cra');
+const fs = require("fs");
 const path = require('path');
+const webpack = require('webpack');
 
-module.exports = override(
-  removeModuleScopePlugin(),
-  babelInclude([path.resolve('src'), path.resolve(__dirname, '../plugins')])
-);
\ No newline at end of file
+const pluginPackagePrefix = "@malwarefront-plugin";
+const pluginsIndexFile = path.resolve(require.resolve("@malwarefront/extensions"), '..', 'plugins.js');
```

Comment:
A bit confused about this. It suggests that `@malwarefront/extensions` is an `npm` module, but in that case we resolve `node_modules/plugins.js` which is concearning. If that's not the case then I'd go from `__dirname` and `path.join` my way to the specified file.

Reply:
`@malwarefront/extensions` is `npm` module and `require.resolve` returns its entrypoint (`node_modules/@malwarefront/extensions/index.js`). Unfortunately Node.js doesn't have function to return just path to the module instead of entrypoint file, so we need to `..` from `index.js`

Reply:
Hmmm, I did not expect it to return entrypoint. Maybe `require.context`?

Reply:
It's not that "require" if you mean https://webpack.js.org/guides/dependency-management/, we're operating in Node.js environment.

https://nodejs.org/api/modules.html#modules_require_resolve_request_options

Reply:
Btw. don't be fooled by `paths` - `paths` option returns something similar to `sys.path` in Python

Reply:
Ah, right. I guess it's the only way :but_at_what_cost: